sdk/nestjs/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "@devcycle/nestjs-server-sdk",
+    "version": "0.0.1",
+    "description": "The DevCycle NestJS Server SDK used for feature management.",
+    "license": "MIT",
+    "dependencies": {
+        "@devcycle/nodejs-server-sdk": "^1.18.3",
+        "@devcycle/types": "^1.9.0",
```

Comment:
all these errors are from the Nx dependency checker lint rule. It basically doesn't work properly most of the time. If you can figure out why it thinks these dependencies aren't used then that would be best, but otherwise you can ignore specific packages from being checked in the `.eslintrc` config for this sdk